build.gradle

```diff
@@ -67,8 +67,8 @@ publishing {
         maven {
             url = "https://s01.oss.sonatype.org/service/local/staging/deploy/maven2/"
             credentials {
-                username = ossrhUsername
-                password = ossrhPassword
+                username = 'ossrhUsername'
+                password = 'ossrhPassword'
```

Comment:
This change is incorrect. Because ossrhUsername and ossrhPassword are variabled defined in gradle.properties.
Ref: https://docs.gradle.org/current/userguide/build_environment.html